ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_node_type_l3_interfaces.schema.yml

```
@@ -127,6 +127,11 @@ $defs:
         raw_eos_cli:
           type: str
           description: EOS CLI rendered directly on the interface in the final EOS configuration.
+        flow_tracking:
+          type: dict
+          $ref: "eos_designs#/$defs/flow_tracking_link"
+          description: |
+            Configures flow-tracking on the interface. Overrides `fabric_flow_tracking` setting.
```

```suggestion
            Configures flow-tracking on the interface. Overrides `fabric_flow_tracking.l3_interfaces` setting.
```

---

seems it was overwrritten ..
```suggestion
            Configures flow-tracking on the interface. Overrides `fabric_flow_tracking.l3_interfaces` setting.
```